version.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://raw.githubusercontent.com/dotnet/Nerdbank.GitVersioning/master/src/NerdBank.GitVersioning/version.schema.json",
-  "version": "1.1",
+  "version": "2.2",
```

Comment:
why a such bimp in versions?

Reply:
I took this as meaning the schema version for `Nerdbank`. My mistake... reverting.